### ISSUE TITLE
Failed to delete [package]: Does not exist

### ISSUE THREAD

**Issue by green1139** [atom/atom#12057]
Still persists in Atom 1.9.0-dev

```
atom --version
Atom    : 1.9.0-dev-8910711
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```
```
apm --version
apm  1.9.2
npm  2.13.3
node 0.10.40
python 2.7.10
git 2.8.1
```
```
uname -rs
Darwin 16.0.0
```

**Comment by thomasjo** [comment]
Please fill in a [proper issue report](https://raw.githubusercontent.com/atom/atom/master/ISSUE_TEMPLATE.md) (note that this template is used by default when creating an issue, so I can only assume you deliberately removed template). As it stands right now, this issue is non-actionable and will be closed to reduce [the deluge of issues](http://blog.atom.io/2016/04/19/managing-the-deluge-of-atom-issues.html) unless the required information is provided in a reasonable amount of time.

**Comment by Ben3eeE** [comment]
This might be referring to an issue issue was fixed in Atom 1.9 so it might not be fixed in your current dev version. Please update to the latest master and try again or try the 1.9 beta.
